cpp/ql/src/experimental/Security/CWE/CWE-703/FindIncorrectlyUsedExceptions.qhelp

```diff
@@ -7,21 +7,17 @@
 
 
 </overview>
-<recommendation>
 
-<p>We recommend that you handle exceptions inside the destructor.</p>
-
-</recommendation>
 <example>
-<p>The following example demonstrates erroneous and fixed methods for using exceptions.</p>
+<p>Finding for places of incomplete use of objects, type exception.</p>
 <sample src="FindIncorrectlyUsedExceptions.cpp" />
 
 </example>
 <references>
 
 <li>
   CERT CPP Coding Standard:
-  <a href="https://wiki.sei.cmu.edu/confluence/display/cplusplus/DCL57-CPP.+Do+not+let+exceptions+escape+from+destructors+or+deallocation+functions">DCL57-CPP. Do not let exceptions escape from destructors or deallocation functions</a>.
+  <a href="https://wiki.sei.cmu.edu/confluence/display/java/ERR00-J.+Do+not+suppress+or+ignore+checked+exceptions">ERR00-J. Do not suppress or ignore checked exceptions</a>.
```

Comment:
This page is about the Java language.  Please would you replace it with an equivalent C/C++ link?